features/step_definitions/common_steps.rb

```diff
@@ -336,6 +336,24 @@
   end
 end
 
+Then /^I see an entry in the '(.*)' table with:$/ do |table_heading, table|
+  result_table_rows = get_table_rows_by_caption(table_heading)
+  expected_row = table.rows[0]
+  for row in result_table_rows
+    result_columns = row.all('td')
+    match = true
+    for value, index in expected_row.each_with_index
+      if match and value != '<ANY>' and value != result_columns[index].text
```

Comment:
if match is overwritten to true by us do we need to check for it in the if statement?